k-distribution/src/main/scripts/bin/krun

```diff
@@ -133,6 +134,9 @@ $KRUN options:
                            if the user says "$KRUN --parser cat foo.kore", then
                            \`cat foo.kore\` is invoked and the result on stdout
                            is used as the value of \$PGM as a KORE term.
+      --no-pattern         Skip kore-match-disjunction invokation when --search
```

Comment:
Let's change this text slightly. I would say instead "skip pattern matching the result of searches on the llvm backend against the default search pattern"